docs/book/quick-start.md

```diff
@@ -1,211 +1,95 @@
 # Quick Start
 
-Now that you have basic knowledge of applications, modules, and how they are
-each structured, we'll show you the easy way to get started.
+In this example, `/hello/world?message=welcome` will display a page containing the message provided through the URL.
+This requires several steps:
+
+- Create a controller to obtain the message from the URL, and pass it as a variable to the view
+- Create a view to display a page containing the message
+- Create a route to match the URL to the controller
 
 ## Install the Laminas MVC Skeleton Application
 
-The easiest way to get started is to install the skeleton application via
-Composer.
+An easy way to get started is to install the skeleton application via Composer.
```

Comment:
Pretty sure I can remember that I've somewhere read that the word "easy" is to be avoided as just because it is easy for the writer of this message does not mean it is easy for the reader. But thats just a single sidenote here.
Overall, good improvement. Thanks!

Reply:
It is no longer a recommended way but instead it is required way. We no longer provide laminas-loader rules nor we endorse its usage.

Also, generally, any mentions of setting up autoloader rules via `Module::getAutoloaderConfig()` config should be removed and replaced with instructions for composer. 
Module manager autoloader is not registered by skeleton [laminas/laminas-mvc-skeleton /config/application.config.php#L15-L16](https://github.com/laminas/laminas-mvc-skeleton/blob/3b83a32d280854ceb4f7602cdf37d2ba241a0115/config/application.config.php#L15-L16) and Application module provided by skeleton does not provide autoloader config.

Reply:
> We no longer provide laminas-loader rules nor we endorse its usage.

That's not the point I was trying to make. You are right, all the old stuff has to go and must not appear in the documentation at all!
But we should not write that it is the "required way", because that sounds a bit negative.

Reply:
I was thinking along dropping "The recommended" part entirely, since it implies there is another way and we do not really support non-composer installations anymore.

Something more assertive. 
```md
Before creating the application ensure the requirements:

- Install PHP 7.4 or newer
- php extensions needed for skeleton install?
- [Install Composer 2.0 or newer](https://getcomposer.org/doc/00-intro.md#installation-linux-unix-osx).
```

Reply:
Yes, here: https://github.com/orgs/laminas/teams/documentation/discussions/1
However, I thought that we needed to avoid "easiest", not "easy". I don't think that it makes sense to avoid words such as "easy" or "good", but it makes sense to avoid "easiest", "best" or "very easy". That's how I interpret that document.